core/Plugin/Menu.php

```diff
@@ -30,16 +30,6 @@
  */
 class Menu
 {
-    protected $module = '';
```

Comment:
See `urlForAction()`. It calls `$this->getModule()` twice in that function. Meaning if it is cached it would be only done once per plugin. When this function `urlForAction()` is called once per plugin, we save one call to `getModule()`. If that method is called 2 times per plugin, we save 3 called to `getModule()` etc.  If 50 plugins call `urlForAction()` once, it saves 50 called in the end since `getModule()` would be executed 100 times. 

Anyway, it is not so important as said if it is not good for DI.